platforms/hyperledger-fabric/configuration/user-certificate.yaml

```diff
@@ -0,0 +1,68 @@
+# This playbook manages user certificates for peers using existing Fabric-CA deployment on k8s
+# The Kubernetes clusters should already be created and the infomation to connect to the
+# clusters be updated in the network-user-certificate.yaml file that is used as an input to this playbook
+###########################################################################################
+# To Run this playbook from this directory, use the following command 
+# (network-user-certificate.yaml also in this directory)
+# ansible-playbook user-certifiate.yaml -e "@./network-user-certificate.yaml"
+############################################################################################
+
+---
+  # This will apply to ansible_provisioners. /etc/ansible/hosts should be configured with this group
+- hosts: ansible_provisioners
+  gather_facts: no
+  tasks:
+    # Check if CA server is available
+    - name: "Waiting for the CA server to be created in {{ item.name | lower }}-net"
```

Comment:
[nitpick] change the wordings from `created` to something like `Checking if CA server exists`. There is no previous role that brings up the CA server for this playbook.